frame/democracy/src/lib.rs

```diff
@@ -811,7 +726,7 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::fast_track())]
 		pub fn fast_track(
 			origin: OriginFor<T>,
-			proposal_hash: T::Hash,
+			proposal_hash: H256,
```

Comment:
Can you clarify your reasoning for choosing a fixed hashing algorithm vs keeping things generic?

Is it that it would be too much code to make preimages and some of the primitives handle this generically?

It seems it would ultimately be possible to make this stuff work with some kind of `HashOfPreimage<T>`

Reply:
We could make it work generically, but realistically the hash length (and probably algorithm) will not change in the lifetime of this API and it makes it quite substantially easier to use/maintain.